apps/ios/Shared/Views/ChatList/ChatListView.swift

```diff
@@ -55,6 +55,22 @@ struct ChatListView: View {
         .onChange(of: chatModel.appOpenUrl) { _ in connectViaUrl() }
         .onAppear() { connectViaUrl() }
         .onDisappear() { withAnimation { userPickerVisible = false } }
+        .refreshable {
+            AlertManager.shared.showAlert(Alert(
+                title: Text("Reconnect?"),
+                message: Text("Reconnect all relays"),
```

Comment:
Relays may be unclear for many users
```suggestion
                message: Text("Reconnect all contacts"),
```

Reply:
contacts and groups?